src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +238,57 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| Observed transverse relaxation rate map      | R2starmap | In seconds<sup>-1</sup> (1/s). R2-star maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                       |
+| Proton density map                           | PDmap     | In arbitrary units (arbitrary). PD maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                           |
+| Magnetization transfer ratio map             | MTRmap    | In arbitrary units (arbitrary). MTR maps are REQUIRED to use this suffix regardless of the method used to generate them. MTRmap intensity values are RECOMMENDED to be represented in percentage in the range of 0-100%.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Magnetization transfer saturation map        | MTsat     | In arbitrary units (arbitrary). MTsat maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                        |
+| Homogeneous (flat) T1-weighted MP2RAGE image | UNIT1     | In arbitrary units (arbitrary). UNIT1 images are REQUIRED to use this suffix regardless of the method used to generate them. Note that although this image is T1-weighted, regions without MR signal will contain white salt-and-pepper noise that most segmentation algorithms will fail on. Therefore, it is important to dissociate it from from `T1w`.                                                                                                                                                        |
```

Comment:
If this is a T1w image (with white noise added), shouldn't it be in the non-parametric images table?

Reply:
I agree. 358370c moves `UNIT1` to the non-parametric table and adds a link to the respective note in the appendix.